src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
```

Comment:
Please don't add redundant empty lines to your code. We don't need them after class declaration or method signature. You can remove the empty line at the start of the `main` method.